### PR TITLE
[feat] save generated images to temp

### PR DESCRIPTION
**Pull request by ghost**
fügt nur eine Methode hinzu, welche ein BufferedImage als Bild speichern kann

### CODE REVIEW

File: src/main/java/github/weichware10/util/Files.java [hunk 1]
```diff
@@ -10,6 +12,9 @@
 import java.net.URL;
 import java.util.Arrays;
 import java.util.List;
+
+import javax.imageio.ImageIO;
```
Comment (weichbot10) [inline src/main/java/github/weichware10/util/Files.java:16]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'javax.imageio.ImageIO'

File: src/main/java/github/weichware10/util/Files.java [hunk 2]
```diff
@@ -10,6 +12,9 @@
...
+
 import org.apache.commons.io.FileUtils;
```
Comment (weichbot10) [inline src/main/java/github/weichware10/util/Files.java:18]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'org.apache.commons.io.FileUtils'

File: src/main/java/github/weichware10/util/Files.java [hunk 3]
```diff
@@ -78,6 +83,20 @@ public static String saveImage(String imageUrl) throws MalformedURLException,
         return destName;
     }
 
+    public static String saveGeneratedImage(BufferedImage buffImage, String fileName)
```
Comment (weichbot10) [inline src/main/java/github/weichware10/util/Files.java:86]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.MissingJavadocMethodCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing a Javadoc comment.

File: src/main/java/github/weichware10/util/Files.java [hunk 4]
```diff
@@ -78,6 +83,20 @@ public static String saveImage(String imageUrl) throws MalformedURLException,
...
+            throws IOException, IllegalArgumentException {
+        if (tmpdir == null) {
+            tmpdir = createTempDir();
+        }
+        RenderedImage rendImage = buffImage;
+
+        String destName = tmpdir + "/" + fileName;
+        File file = new File(destName);
+        ImageIO.write(rendImage, "png", file);
+
```
Comment (weichbot10) [inline src/main/java/github/weichware10/util/Files.java:96]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.regexp.RegexpSinglelineCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line has trailing whitespaces.

**Review by ghost — APPROVED** [review]
giphy
)

**Review by Rotwolf — APPROVED** [review]

**Pull request merged**